src/schema/v2/me/user_flags.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  GraphQLError,
+  GraphQLFieldConfig,
+  GraphQLScalarType,
+  Kind,
+  ObjectValueNode,
+  print,
+  ValueNode,
+} from "graphql"
+import Maybe from "graphql/tsutils/Maybe"
+import { camelCase } from "lodash"
+import { ResolverContext } from "types/graphql"
+
+export const UserFlagsType = new GraphQLScalarType({
+  name: "UserFlagsType",
+  description:
+    "The user's user_flags. Values of keys in user_flags can be anything and keys may not be known beforehand.",
+  serialize: (value) => value,
+  parseValue: (value) => value,
+  parseLiteral: (ast, variables) => {
+    if (ast.kind !== Kind.OBJECT) {
+      throw new TypeError(
+        `Object cannot represent non-object value: ${print(ast)}`
+      )
+    }
+
+    return parseObject("Object", ast, variables)
+  },
+})
+
+export const userFlagsResolver = async (root, _options, loaders) => {
+  const { meLoader } = loaders
+  const { user_flags: userFlags } = root
+  if (!meLoader) {
+    throw new Error("You need to be signed in to perform this action")
+  }
+
+  const res = {}
+  const keys = Object.keys(userFlags)
+  keys.forEach((key) => {
+    res[camelCase(key)] = userFlags[key]
+  })
+  return res
```

Comment:
This would definitely make it shorter but `root` is any, `userFlags` is any.
I feel more confident in `Object.keys(anythingButUndefinedandNull)` than in `mapKeys(anything as {[key: string]: any}, (_v, k) => camelCase(k))`. What do you think?

Reply:
I'm not sure if I understand your concern. `mapKeys(userFlags, (v, k) => camelCase(k))` doesn't require any type casting as well.

Reply:
I'd need to typecast `userFlags`.

Reply:
Oh yeah, you're right! I didn't see the typescript warning...

That's a funny behavior of `mapKeys`. It seems that if you pass an `object` of type `any` it types the key as `number` in the iterator.